warehouse/malware/checks/base.py

```diff
@@ -20,17 +21,28 @@ def __init__(self, db):
         self._load_check_id()
         self._verdicts = []
 
+    @classmethod
+    def prepare(cls, request, obj_id):
+        # TODO(ww): Document.
+        kwargs = {}
+
+        if cls.hooked_object == "File":
+            path = request.db.query(File.path).filter(File.id == obj_id).one()
```

Comment:
I believe you can use `get(obj_id)` here instead of `filter(File.id == obj_id).one()`